index.ts

```diff
@@ -160,6 +160,11 @@ function delegate<
 		}
 	};
 
+	// Drop unsupported `once` option https://github.com/fregante/delegate-it/pull/28#discussion_r863467939
+	if (typeof options === 'object') {
+		delete options.once;
```

Comment:
> Should this PR also change the accepted options types? E.g.

Yes, we don't accept `once` so the types are currently incorrect. Then you can probably avoid the error with:

```suggestion
		delete (options as AddEventListenerOptions).once;
```